src/textual/screen.py

```diff
@@ -284,7 +284,6 @@ def _watch_stack_updates(self):
 
     def refresh_bindings(self) -> None:
         """Call to request a refresh of bindings."""
-        self.log.debug("Bindings updated")
```

Comment:
This log is super noisy and low value, and as far as I know there is no way of switching it off without also disabling debug logs in my own code.